src/ploneintranet/workspace/workspacecontainer.py

```diff
@@ -15,3 +15,12 @@ class WorkspaceContainer(Container):
     A folder to contain WorkspaceFolders
     """
     grok.implements(IWorkspaceContainer)
+
+try:
```

Comment:
Alternatively, this could be a dexterity behaviour in ploneintranet.attachments. It could then be applied to a workspace using a policy egg that brings the two packages together. See https://github.com/ploneintranet/ploneintranet.attachments/issues/4